requirements.txt

```diff
@@ -49,3 +49,5 @@ google-auth==1.30.0
 pandas==1.3.0
 pyarrow==4.0.1
 more-itertools==8.8.0
+librabbitmq
```

Comment:
this package hasn't been updated since 2018. Did you see https://pypi.org/project/librabbitmq-fork? This is updated recently, and also has wheels for more recent python versions, so we wouldn't have to build it from source.
The github page of librabbitmq-fork is forked from celery/librabbitmq, and all changes have actually been merged back to the celery repo too, so I think that for now using the -fork package from pypi is the better option

Reply:
Done.